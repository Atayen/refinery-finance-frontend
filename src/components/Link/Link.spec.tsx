import React from 'react';
import { shallow } from 'enzyme';
import Link from './index';

describe('Link', () => {
  it('Renders link to Google', () => {
    const link = shallow(<Link href="http://google.com">Link to Google</Link>);
    expect(link).toMatchSnapshot();
  });
});
