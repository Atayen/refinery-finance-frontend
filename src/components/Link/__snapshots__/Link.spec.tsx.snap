// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link Renders link to Google 1`] = `
<a
  href="http://google.com"
>
  Link to Google
</a>
`;
